pkg/worker/mount.go

```diff
@@ -0,0 +1,91 @@
+package worker
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/beam-cloud/beta9/pkg/common"
+	"github.com/beam-cloud/beta9/pkg/storage"
+	"github.com/beam-cloud/beta9/pkg/types"
+)
+
+type StorageManager struct {
+	mountPointPaths *common.SafeMap[[]string]
```

Comment:
should this be a map of container ID to the mount objects themselves? that way you can clear them by container ID. Also then its not mountPointPaths, it's just `containerMounts`